app/admin/golem/components/BadMatchModal.tsx

```
@@ -0,0 +1,150 @@
+            disabled={saving}
+            className="text-white/40 hover:text-white transition-colors disabled:opacity-30"
+          >
+            <X className="h-5 w-5" />
+          </button>
+        </div>
+
+        <p className="text-sm text-white/60 mb-4 line-clamp-2">{jobTitle}</p>
+
+        {/* Tag chips */}
+        <div className="mb-4">
+          <label className="text-xs text-white/50 mb-2 block">Why is this a bad match?</label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)